docs/console/company-configuration/clusters-management/cluster-setup.mdx

```diff
@@ -158,6 +159,31 @@ You can choose between the automatic procedure and the manual one.
         - {{NAME_OF_THE_SERVICE_ACCOUNT_USED_FOR_THE_DEPLOY}}
   ```
 
+  #### Monitoring custom resources from the Console
+
+  Following these guidelines for configuring custom resources. 
+  We need to make sure that the console has the correct permissions 
+  to display the information to users.
+
+  Here is an example to show how to give the correct permissions to custom resource definitions:
+  
+  ```yaml
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  # ...
+  rules:
+    # For each custom resource definitions we need to add:
+    - apiGroups:
+        - "<custom-resource-definitions.group>"
+      resources:
+        - "<custom-resource-definitions.names.plural>"
+      verbs:
+        - get
+        - list
+  ```
+
+  After adding the necessary permissions follow this [guide](/console/design-your-projects/custom-resources/custom-resources.md) to see your custom resources inside the console
+
   ### ClusterRoleBindings
 
   Finally, we **must** associate the roles with the service account using the `ClusterRoleBindings` that can be created using the following template:
```

Comment:
```suggestion
  Finally, make sure to associate the roles with the service account using the `ClusterRoleBindings` that can be created using the following template:
```

Reply:
must feels like too much on a documentation, plus there is inconsistency between 'we' and 'you'. I'd use always 'you', this is a guide for the user